marathon_acme/service.py

```diff
@@ -194,5 +195,41 @@ def _issue_certs(self, domains):
                 len_domains=len(domains), domains=domains)
         else:
             self.log.debug('No new domains to issue certificates for')
-        return gatherResults(
-            [self.txacme_service.issue_cert(domain) for domain in domains])
+        return gatherResults([self._issue_cert(domain) for domain in domains])
+
+    def _issue_cert(self, domain):
+        """
+        Issue a certificate for the given domain.
+        """
+        def errback(failure):
+            # Don't fail on some of the errors we could get from the ACME
+            # server, rather just log an error so that we can continue with
+            # other domains.
+            failure.trap(txacme_ServerError)
+            acme_error = failure.value.message
+
+            # FIXME: The acme error code stuff is a mess pre- the unreleased
+            # 0.10 version. Update this to use the 'code' attribute when the
+            # new acme library is released. The 'urn:acme:error' string doesn't
+            # seem to be in a constant anywhere either...and will also change
+            # (in some cases) in the new acme library version.
+            if not acme_error.typ.startswith('urn:acme:error:'):
```

Comment:
Can we just look at `acme_error.typ.split(':')[-1]` here? That should get us past all the various differences.